src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/BreakpointComplications.java

```diff
@@ -313,6 +314,16 @@ boolean isDupAnnotImprecise() {
             return dupAnnotIsImprecise;
         }
 
+        /**
+         * @return may be null if the complication is not imprecise
+         */
+        public SimpleInterval getImpreciseDupAffectedRefRange() {
+            if (dupAnnotIsImprecise)
```

Comment:
Do you need this logic in the getter? Why not just keep the field null if it's not an imprecise dup, ie. handle this in the constructor?